File: bubblesort.py
def bubble_sort(array):
    n = len(array)
    for i in range(n - 1):
        for j in range(n - i - 1):
            if array[j] > array[j + 1]:
                array[j], array[j + 1] = array[j + 1], array[j]

array = list(map(int, input("Enter your array: ").split()))

bubble_sort(array)

print("Sorted array is:", *array)

File: bubblesort.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let array = [];

rl.question('Enter your array: ', (input) => 
{
    array = input.split(' ').map(Number);

    for(let i = 0; i < array.length - 1; i++) 
    {
        for(let j = 0; j < array.length - i - 1; j++) 
        {
            if(array[j] > array[j + 1]) 
            {
                let temp = array[j];
                array[j] = array[j + 1];
                array[j + 1] = temp;
            }
        }
    }

    console.log('Sorted array is: ' + array.join(' '));

    rl.close();
});
